src/handlebars/helpers.ts

```diff
@@ -14,9 +14,23 @@ export function registerHandlebarsHelpers() {
     }
     return options.inverse(this);
   });
+  handlebars.registerHelper('cellCanBePlayed', function (cellValue, readyToPlay, currentPlayerTurnToPlay, options) {
+    if (cellValue === "empty" && readyToPlay && currentPlayerTurnToPlay) {
+      return options.fn(this);
```

Comment:
this is really obscure. Perhaps you need TS types & JS doc to show how to use it